src/store/modules/accounts/hdWallet.js

```diff
@@ -86,38 +86,40 @@ export default {
     sign({ dispatch }, data) {
       return dispatch('signWithoutConfirmation', { data });
     },
-    async signTransaction({ dispatch, rootGetters }, {
+    async signTransaction(context, {
       txBase64,
       options: { modal = true, app = null },
     }) {
+      const { nodeNetworkId } = useAeSdk({ store: context });
```

Comment:
Won't there be problems with `state` being not `rootState` and getters being not `rootGetters`?

Reply:
Totally right. I got inspired by the `isAccountUsed` action which now I think has a bug. I asked @subhod-i to remove it as the adapters has the same logic.